src/lib/GameList/GameListCard.svelte

```diff
@@ -50,7 +59,7 @@
 		margin-bottom: 0;
 	}
 
-	.label-container {
+	/* .label-container {
```

Comment:
Is deze comment nodig?

Reply:
Hierin staat styling voor de `new` card die bovenaan staat. Weet niet of dit behouden moest worden of dat dit weg kon. Zal ik het dus gewoon verwijderen?

Reply:
Zal hem voor nu gewoon weghalen! kan altijd weer opgezocht worden.